### DISCUSSION TITLE
Add FreeBSD swc support

### DISCUSSION THREAD

**Discussion by dcschmid** [vercel/next.js#34532] [Ideas]
### Describe the feature you'd like to request

Hey dear Vercel Team,

is it possible to activate SWC support for FreeBSD systems? Personal I use an FreeBSD-based developer Laptop. 

### Describe the solution you'd like

After a search on the internet, I found out there is a FreeBSD core SWC package from the SWC Team. https://www.npmjs.com/package/@swc/core-freebsd-x64

Is it possible to build in this package in the Next Framework itself? 

Would be very helpful to use it on FreeBSD too.

### Describe alternatives you've considered

Yes, I can use the babel solution, but swc would be better :-)

**Comment by toobr** [comment]
Is it currently possible to use this package in a Next build?
